### PR TITLE
Decommission guui

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Decommission package `guui`.

## Why?

Needed Refactoring. ( https://trello.com/c/psXEfcWL/729-remove-pasteup-and-guui )

## Details

Before:
`import { Dropdown } from '@guardian/guui/components/Dropdown/Dropdown';`
After:
`import { Dropdown } from '@frontend/web/components/Dropdown';`

Before:
`import { Caption } from '@guardian/guui/components/Caption/Caption';`
After:
`import { Caption } from '@frontend/web/components/Caption';`
`import { Caption } from '@frontend/amp/components/Caption';`

Before
`import { Byline } from '@guardian/guui/components/Byline/Byline';`
After:
`import { Byline } from '@frontend/web/components/Byline';`

Before:
`import { BigNumber } from '@guardian/guui';`
After:
`import { BigNumber } from '@frontend/web/components/BigNumber/BigNumber';`

### CODE REVIEW

**Comment by shtukas** [comment]
#IgnoreCodeClimate

File: packages/frontend/package.json [hunk 1]
```diff
@@ -1,58 +1,57 @@
...
-    "@types/html-minifier": "^3.5.2",
-    "@types/jsdom": "^11.0.6",
-    "@types/lodash.get": "^4.4.4",
-    "@types/node": "^10.12.0",
-    "@types/raven-js": "^3.10.0",
-    "@types/react": "^16.4.11",
-    "@types/react-dom": "^16.0.7",
-    "amphtml-validator": "^1.0.23"
-  }
+    "name": "@guardian/frontend-rendering",
```
Comment (nicl) [inline packages/frontend/package.json]: Ah this file is annoying to edit as it gets reformatted automatically in a way that is different from the yarn/npm default.

File: packages/frontend/web/components/BigNumber/0.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="34" height="40">
```
Comment (nicl) [inline packages/frontend/web/components/BigNumber/0.svg]: All of these probably should live with the pasteup icons that I've moved to the frontend package under `static/` but that PR isn't merged yet so I'll add that once this is merged (or when they both are).

File: packages/frontend/web/components/BigNumber/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Number
+======
+
+### Import
+```js
+import { Number } from '@guardian/guui';
```
Comment (nicl) [inline packages/frontend/web/components/BigNumber/README.md]: is this a @guui reference?

Comment (shtukas) [reply packages/frontend/web/components/BigNumber/README.md]: Oh thanks for catching this.

**Pull request merged**